src/Translation/locales/ja.json

```diff
@@ -266,7 +266,7 @@
     "expired_at": "{time}に締切済み",
     "expires_in": "締切：{time}",
     "expires_in_more_than_years": "締切まであと{years}年以上",
-    "for": "for",
+    "for": " ",
```

Comment:
Just to double check. Is this ok?

Reply:
Yes. I see It's only used in a sentence that "you're about buying X,Y for 000MANA". It can't be translated "for" as itself in Japanese. It's understandable even without it:)